### ISSUE TITLE
Wrong File URL for Video Files using DigitalOcean Spaces in Wordpress Media Library

### ISSUE THREAD

**Issue by ChristophJeworutzki** [Interfacelab/ilab-media-tools#189]
**Describe the bug**
Getting wrong File URL for Video Files in Wordpress Media Library using DigitalOcean Spaces

**To Reproduce**
1. Enable Cloud Storage
2. Set Storage Provider to "DigitalOcean Spaces"
3. Set Provider Settings (Access Key, Secret, Bucket, … )
4. Enable Upload Video Files
5. Save Changes
6. Go to Media Library
7. Upload a Video (web supported file format e.g mp4)
8. Click the Video
9. Get "Media error: Format(s) not supported or source(s) not found"
10. See File URL in the Attachment Info: 'fra1.digitaloceanspaces.com/my-bucket-name/…' insted of 'https://my-bucket-name.fra1.digitaloceanspaces.com/…'

**Expected behavior**
Correct File URL for Video Files (same as for Image Files)
"https://my-bucket-name/fra1.digitaloceanspaces.com/..."

**Screenshots**
<img width="1388" alt="Screenshot 2022-10-19 at 17 25 00" src="https://user-images.githubusercontent.com/30624279/196735817-47ff2521-7c13-4b17-9ae5-3481e7d27da5.png">
<img width="1383" alt="Screenshot 2022-10-19 at 17 25 06" src="https://user-images.githubusercontent.com/30624279/196736163-4c3f6899-9de7-46b5-80e5-4255c349c13a.png">

**Server (please complete the following information)**
- Operating System: _Ubuntu 20.04.5 LTS_ 
- PHP Version _PHP_ 7.4
- Web Server: _NGNIX_
- Using PHP-FPM? _Yes_
- Using WordPress Cron? _No_

**How did you install the plugin?**
via composer

**Comment by ChristophJeworutzki** [comment]
The funny thing is that it works for small and short Video Files < 5 MB and < 10s

**Comment by jawngee** [comment]
Do you have mux or the mux player enabled?  I tried to replicate this and couldn't.  Size wouldn't have any bearing on anything at all.

**Comment by ChristophJeworutzki** [comment]
Yes. I have mux and mux player enabled. But still happends if I turn off both.

**Comment by jawngee** [comment]
I found the issue.  Not sure why it exists.  Will dig further tomorrow.

**Comment by ChristophJeworutzki** [comment]
> I found the issue. Not sure why it exists. Will dig further tomorrow.

Nice one! Let me know if you need anything future information!

**Comment by ChristophJeworutzki** [comment]
> I found the issue.  Not sure why it exists.  Will dig further tomorrow.

Sorry for bothering you. Just wanted to know if you have any updates on this? Appreciate your help!

**Comment by jawngee** [comment]
Should be fixed in this latest release.  LMK if u run into any problems.

**Comment by jawngee** [comment]
@ChristophJeworutzki did that fix things on your end?

**Comment by ChristophJeworutzki** [comment]
@jawngee Yes, I can confirm that it has been fixed! Thank you! 🙏

**Comment by satodu** [comment]
Hi,

How was it solve @jawngee  ?

Thanks nad regards
